tests/kernel_performance/install_qatestset.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary: setup performance run environment
+# Maintainer: Joyce Na <jna@suse.de>
+
+package install_qatestset;
+use base "y2logsstep";
+use strict;
+use warnings;
+use utils;
+use testapi;
+
+sub install_pkg {
+    my $qa_server_repo = get_var('QA_REPO');
+    assert_script_run("hostname > /etc/hostname");
+    zypper_call("--non-interactive rr qa-ibs");
+    zypper_call(
+        "--non-interactive --no-gpg-check -n ar -f '$qa_server_repo' qa-ibs");
+    zypper_call("-n --no-gpg-check ref");
+    zypper_call("-n install qa_testset_automation");
+}
+
+sub setup_environment {
+    my $runid             = get_var('QASET_RUNID');
+    my $mitigation_switch = get_var('MITIGATION_SWITCH');
+    assert_script_run(
+        "/usr/share/qa/qaset/bin/deploy_performance.sh SLE-12-SP4 $runid $mitigation_switch"
```

Comment:
I think the hard code 'SLE-12-SP4' should be avoid, as the test is applicable for all releases

Reply:
Updated.